src/ruby/LICENSE

```diff
@@ -0,0 +1,95 @@
+Copyright (c) 2013 SymPy Development Team
```

Comment:
We might need it if the user decides only to install the gem. This file is included along all the other files of the gem [here](https://github.com/abinashmeher999/symengine/blob/ruby_file_structure/src/ruby/symengine.gemspec#L9). If there's a different license file I can add the path to that in place of this. Or have license at the beginning of each source file.